pkg/collector/corechecks/net/networkv2/network.go

```diff
@@ -145,8 +145,15 @@ func (c *NetworkCheck) Run() error {
 				c.submitProtocolMetrics(sender, counters[protocol])
 			}
 		}
+		if tcpStats, ok := counters["Tcp"]; ok {
+			if val, ok := tcpStats.Stats["CurrEstab"]; ok {
+				sender.Gauge("system.net.tcp.current_established", float64(val), "", nil)
+			}
+		}
 	}
 
+	submitInterfaceSysMetrics(sender)
```

Comment:
[this](https://github.com/DataDog/integrations-core/blob/master/network/datadog_checks/network/check_linux.py#L330-L348) was not implemented earlier and we missed some default metrics